test/runtests.jl

```diff
@@ -24,3 +24,13 @@ end
 @testset "manual examples" begin
   include("doctest.jl")
 end
+
+@testset "JuliaInterface tests" begin
+  GAP.Wrappers.Read(joinpath(dirname(dirname(pathof(GAP))), "pkg/JuliaInterface/tst/testsetup.g"))
+  GAP.Globals.TestDirectory(GAP.Globals.dir, GapObj(Dict(:exitGAP => false, :testOptions => Dict(:compareFunction => GAP.Globals.compare)); recursive=true))
```

Comment:
`TestDirectory` returns `true` or `false`, thus I think we can call `@test GAP.Globals.TestDirectory(...)`.